packages/react-divider/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@fluentui/react-divider",
+  "version": "0.1.0",
+  "description": "React components for building web experiences",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
```

Comment:
@ling1726 should probably look over this list of deps to ensure it's still what we want for converged.

Reply:
Can we discussion what is needed and how to approach, these are from the create-package script.

Reply:
Part of the issue here is that create-package has been updated a lot recently, and the template has changed since you ran it. That's why I asked Ling to look over things manually in this instance.

Reply:
This seems to be a broader conversation that needs to happen across all converged components.. can we move this to a new PR?